### DISCUSSION TITLE
Question: TabView fails to show content Frame & apply current Dark Theme

### DISCUSSION THREAD

**Discussion by sigmarsson** [microsoft/microsoft-ui-xaml#7414] [Q&A]
I create and navigate to a new page associated to the ```TabViewItem``` but the view inside isn't shown until I open a new ```TabViewItem``` and change back to the first tab, which initially displayed no content. Here is how I create each tab : 

```
        private TabViewItem NewTab(string header = null)
        {
            ActiveDocument = new()
            {
                Header = header ?? HeaderIndex.ToString(),
                IconSource = new SymbolIconSource
                {
                    Symbol = Symbol.Document
                }
            };

            ActiveDocument.Content = NavigateTo();

            return ActiveDocument;
        }

        private static Frame NavigateTo()
        {
            Frame frame = new();

            frame.Navigate(typeof(Views.ChartRelayView));

            return frame;
        }
```

**Comment by sigmarsson** [comment]
The Element Selector reveals the view is there in the ```TabViewItem``` but not displayed correctly until I open a new tab navigate there and then back to the first tab. 

![image](https://user-images.githubusercontent.com/43340959/172142625-28eee913-7825-47d1-9411-b1a4433ba038.png)

**Comment by sigmarsson** [comment]
This footage demonstrates what I described above + exhibits the failure that the ```TabView``` is unable to apply the current dark theme until I maximize the window. The tabs are visibale and apply the dark theme from the moment I maximized the window. Footage is ZIPed due to size restriction. 

At the begingn you can see that the first tab content is not visible until I change tabs. 

[2022-06-06 14-44-53.zip](https://github.com/microsoft/microsoft-ui-xaml/files/8844576/2022-06-06.14-44-53.zip)

**Comment by sigmarsson** [comment]
Footage: 
[2022-06-06 23-35-28.zip](https://github.com/microsoft/microsoft-ui-xaml/files/8848115/2022-06-06.23-35-28.zip)

**Comment by sigmarsson** [comment]
Now i tested this again, ```TabView``` fails to apply theme even upon maximize/unmaximize window . Please see pictures : 


![image](https://user-images.githubusercontent.com/43340959/175514068-78eb70f1-5f50-4a28-816a-ca5968844485.png)

![image](https://user-images.githubusercontent.com/43340959/175514235-0ef7d0c0-5c19-4216-a17e-625103e3719a.png)

**Comment by sigmarsson** [comment]
@bkudiess  can you please explain why you deem it isn't a bug and rather a discussion ?

**Comment by kmgallahan** [comment]
@sigmarsson You named the issue:

> Question: TabView fails to show content Frame & apply current Dark Theme

Starting an issue with "Question:" means it is auto-tagged with the `Question` tag.

@bkudiess blanket converted all issues with the `Question` tag to Discussions, even while it should not have been done for most of them.

Now @ranjeshj [suggests to create new issues](https://github.com/microsoft/microsoft-ui-xaml/discussions/7512#discussioncomment-3223662) to restore any of the ~100 discussions that should not have been converted.

¯\\\_(ツ)\_/¯

**Reply by sigmarsson** [reply]
@ranjeshj you might possess grasp here, what could be the reason ```TabView``` fails to display content until tabs changed ?

**Comment by sigmarsson** [comment]
#7684 is relating to this issue.

**Comment by sigmarsson** [comment]
The ```Pivot``` control works impeccable in ```TabView```'s place. Perhaps there is a bug in the ```TabView``` control.
